### PR TITLE
Fix asciidoc warnings.

### PR DESCRIPTION
**Pull request by yhontyk**
> Read our [Contribution guide](https://github.com/eclipse/che-docs/blob/master/CONTRIBUTING.adoc) before submitting a PR.

### PR Checklist

As the author of this Pull Request I made sure that:

- [x] `vale` has been run successfully against the PR branch
- [x] Link checker has been run successfully against the PR branch
- [x] Documentation describes a scenario that is already covered by QE tests, otherwise an issue has been created and acknowledged by Che QE team
- [x] Changed article references are updated where they are used (or a redirect has been set up on the docs side):
    - [x] Dashboard [branding.json](https://github.com/eclipse/che-dashboard/blob/master/src/components/branding/branding.json)
    - [x] Chectl [constants.ts](https://github.com/che-incubator/chectl/blob/master/src/constants.ts)

### CODE REVIEW

**Review by MichalMaler — APPROVED** [review]

File: modules/administration-guide/partials/proc_installing-image-puller-using-che-operator.adoc [hunk 1]
```diff
@@ -71,7 +71,7 @@ spec:
 $ {orch-cli} get clusterserviceversions
 ----
 
-. The `eclipse-che namespace` contains these deployments: `{image-puller-deployment-name}` and `{image-puller-deployment-id}`.
+. The `eclipse-che namespace` contains these deployments: `{image-puller-deployment-name}` and `{image-puller-operator-repository-name}`.
```
Comment (ibuziuk) [inline modules/administration-guide/partials/proc_installing-image-puller-using-che-operator.adoc]: @boczkowska for me it is not really clear why deployment name is expected to be the repository name

Comment (yhontyk) [reply modules/administration-guide/partials/proc_installing-image-puller-using-che-operator.adoc]: @ibuziuk I believe the deployment here is supposed to be `kubernetes-image-puller-operator`, we have two attributes that store this value:

>image-puller-operator-id: kubernetes-image-puller-operator
>image-puller-operator-repository-name: kubernetes-image-puller-operator

We don't have `image-puller-deployment-id` attribute defined at all. I mean, I could define it but is there a reason for us to store identical information in three attributes? The dowsntream docs aren't affected by these as far as I can see.

Comment (yhontyk) [reply modules/administration-guide/partials/proc_installing-image-puller-using-che-operator.adoc]: @ibuziuk ...but thinking about it, you're right, naming an attribute properly and just defining it seems like a more straightforward way. Reverted this commit and defined the `image-puller-deployment-id` in antora playbook.

Comment (MichalMaler) [reply modules/administration-guide/partials/proc_installing-image-puller-using-che-operator.adoc]: Top-notch

File: antora-playbook.yml [hunk 2]
```diff
@@ -57,6 +57,7 @@ asciidoc:
     image-puller-operator-name: Kubernetes Image Puller Operator
     image-puller-operator-id: kubernetes-image-puller-operator
     image-puller-operator-repository-name: kubernetes-image-puller-operator
+    image-puller-deployment-id: kubernetes-image-puller-operator
```
Comment (ibuziuk) [inline antora-playbook.yml:60]: could you point me where is this property is used?

Comment (yhontyk) [reply antora-playbook.yml:60]: @ibuziuk sure!
https://github.com/eclipse/che-docs/edit/master/modules/administration-guide/partials/proc_installing-image-puller-using-che-operator.adoc#L74

>. The `eclipse-che namespace` contains these deployments: `{image-puller-deployment-name}` and `{image-puller-deployment-id}`.

Comment (ibuziuk) [reply antora-playbook.yml:60]: ack. makes sense

**Review by ibuziuk — APPROVED** [review]

**Review by MichalMaler — APPROVED** [review]

**Pull request merged**